CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.13.7 - 2020-02-14
+
+* Update to package_config `1.9.0` which supports package_config.json
```

Comment:
Note that `packageRoot` is now deprecated in the API?

Reply:
done